exchange/utils.go

```diff
@@ -700,3 +701,42 @@ func getExtBidAdjustmentFactors(requestExt *openrtb_ext.ExtRequest) map[string]f
 	}
 	return bidAdjustmentFactors
 }
+
+func buildXPrebidHeader(bidRequest *openrtb2.BidRequest, revision string) string {
+	req := &openrtb_ext.RequestWrapper{BidRequest: bidRequest}
+
+	pbsRecord := createNameVersionRecord("pbs-go", revision)
+
+	var channelVersionRecord string
+	if reqExt, _ := req.GetRequestExt(); reqExt != nil {
```

Comment:
Please do not ignore errors. Capture the error to a variable and include a check for it in the if clause. Same for the sdk version.

Reply:
Oh yeah, of course... kinda forgot the returned object can be non-nil if there's an error. Thanks for noticing.